compiler/generator/rust/rust_code_container.cpp

```diff
@@ -632,13 +636,56 @@ void RustCodeContainer::generateComputeHeader(int n, std::ostream* fOut, int fNu
 void RustCodeContainer::generateComputeInterfaceHeader(int n, std::ostream* fOut, int fNumInputs,
                                                        int fNumOutputs)
 {
-    // Compute "compute" declaration
     *fOut << "pub fn compute("
           << "&mut self, " << fFullCount << ": usize, inputs: & [& [FaustFloat] ]"
           << ", outputs: & mut[& mut[FaustFloat] ]) {";
     tab(n + 1, *fOut);
 }
 
+void RustCodeContainer::generateComputeFrame(int n)
+{
+    tab(n, *fOut);
+    tab(n, *fOut);
+    *fOut << "pub fn compute_frame(&mut self, inputs: &[&FaustFloat], ";
```

Comment:
Can you use `frame` instead of `compute_frame` to be similar to what C and C++ already generate ?

Reply:
i prefer compute_frame because it makes clearer that it is a variant of compute.